code/modules/voice/voice_packs.dm

```diff
@@ -0,0 +1,150 @@
+#define VOICE_PACKS_FILE "config/monkestation/voice_packs.toml"
+
+/proc/get_voice_pack_sound(voice_pack_obj, group_path, sound_name)
+	var/sound_path = voice_pack_obj[sound_name]
+	if (!sound_path)
+		return null
+	if (group_path)
+		return sound(group_path + "/" + sound_path)
+	else
+		return sound(sound_path)
+
+/proc/gen_voice_packs()
+	if(!fexists(VOICE_PACKS_FILE))
+		log_config("No voice packs file found, generating fallback")
+
+		var/datum/voice_pack/voicepack = new()
+		voicepack.id = "fallback.fallback"
+		voicepack.name = "Fallback"
+		voicepack.group_name = "Fallback"
+		voicepack.sounds = list(
+			sound('sound/effects/adminhelp.ogg'),
+			sound('sound/effects/adminhelp.ogg'),
+			sound('sound/effects/adminhelp.ogg'),
+		)
+		voicepack.max_pitch = VOICE_DEFAULT_MAXPITCH
+		voicepack.min_pitch = VOICE_DEFAULT_MINPITCH
+		voicepack.max_speed = VOICE_DEFAULT_MAXSPEED
+		voicepack.min_speed = VOICE_DEFAULT_MINSPEED
+		voicepack.volume = 1
+		voicepack.hidden = FALSE
+		voicepack.is_goon = TRUE
```

Comment:
couldn't this just be a unique "fallback" subtype (like, `/datum/voice_pack/fallback`) instead of hardcoded var sets?